ctapipe/tools/tests/test_tools.py

```diff
@@ -221,7 +221,7 @@ def test_display_integrator(tmp_path):
     assert (
         run_tool(
             DisplayIntegrator(),
-            argv=shlex.split(f"--f={GAMMA_TEST_LARGE} " "--max_events=1 "),
+            argv=shlex.split(f"--input={GAMMA_TEST_LARGE} " "--max_events=1 "),
```

Comment:
Could you just give the arguments as list? `[f"--input={GAMMA_TEST_LARGE}", "--max_events=1"]`? 

using shlex.split here is not what we want I think. We know, what are the single arguments that need to be quoted together.

Reply:
for all occurences in that file?